src/addToList.js

```diff
@@ -0,0 +1,8 @@
+export default (stock, todo) => {
+  stock.push({
+    description: todo,
+    completed: 0,
+    index: stock.length + 1,
+  });
+  localStorage.setItem('stock', JSON.stringify(stock));
+};
```

Comment:
- [ ] Suppose we have two tasks already and when we refresh the page and then try adding a new task all previous tasks got removed from the localstorage. Please fix this issue.